src/main/scala/nl/knaw/dans/easy/license/Command.scala

```diff
@@ -15,16 +15,92 @@
  */
 package nl.knaw.dans.easy.license
 
-import org.slf4j.LoggerFactory
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, FileOutputStream, OutputStream}
+
 import nl.knaw.dans.easy.license.{CommandLineOptions => cmd}
+import org.slf4j.LoggerFactory
+import rx.lang.scala.{Observable, ObservableExtensions}
+import rx.lang.scala.schedulers.IOScheduler
+import rx.schedulers.Schedulers
+
+import scala.collection.JavaConverters._
+import scala.language.postfixOps
+
+class Command(datasetLoader: DatasetLoader,
+              placeholderMapper: PlaceholderMapper,
+              templateResolver: TemplateResolver,
+              pdfGenerator: PdfGenerator)(implicit parameters: Parameters) {
```

Comment:
As explained in the comment you link to the `Command` object was intended to be only an entry-point, namely the entry point for when you are calling the service from the command line. You have placed the `run` method also here (well, in the `Command` class). You should move that to a generic object, say, `LicenseCreator`, so that only the processing that is specific to the command line interface (such as command line option parsing) stays in `Command`.